src/utils/operation.utils.ts

```diff
@@ -53,37 +53,43 @@ export const mapOperationsToActivities = (address: string, operations: Array<Ope
           isDefined(fa2Parameter.value[0].txs)
         ) {
           contractAddress = target.address;
+          let isFoundAddress = false;
```

Comment:
maybe `isAddressFound`?  and what does this mean?

Reply:
it means that user's account address was not in the sender or target, but in params of fa2 or fa1.2 transfer